libs/admin/src/lib/state/admin.selector.ts

```diff
@@ -0,0 +1,11 @@
+import { createSelector } from '@ngrx/store';
+import { AppStateWithAdmin } from './admin.reducer';
+
+const assetAdminFeature = (state: AppStateWithAdmin) => state.admin;
```

Comment:
I'd drop the `asset` prefix here, the store also doesn't use it.
```suggestion
const adminFeature = (state: AppStateWithAdmin) => state.admin;
```